core/shared/src/main/scala/sigma/serialization/CoreSerializer.scala

```diff
@@ -0,0 +1,68 @@
+package sigma.serialization
+
+import scorex.util.ByteArrayBuilder
+import scorex.util.serialization.{Serializer, VLQByteBufferReader, VLQByteBufferWriter}
+import sigma.data.SigmaConstants
+
+import java.nio.ByteBuffer
+
+abstract class CoreSerializer[TFamily, T <: TFamily] extends Serializer[TFamily, T, CoreByteReader, CoreByteWriter] {
```

Comment:
ScalaDoc missed, what does "Core" means ?

Reply:
Added ScalaDocs for `CoreXXX` classes.